app/src/main/java/com/codingblocks/cbonlineapp/library/LibraryViewFragment.kt

```diff
@@ -63,7 +63,8 @@ class LibraryViewFragment : BaseCBFragment() {
                                 startActivity(
                                     intentFor<PdfActivity>(
                                         CONTENT_ID to item.contentId,
-                                        SECTION_ID to item.sectionId
+                                        SECTION_ID to item.sectionId,
+                                        RUN_ATTEMPT_ID to item.runAttemptId
```

Comment:
why do we need this ?

Reply:
Bhaiya for marking or removing the bookmark functionality, we need run attempt id. Since we are moving to that specific screen, if user wants to remove or add again, runAttempt id is needed